### ISSUE TITLE
Search machine learning/neural network algorithms for image proccessing/ocr

### ISSUE THREAD

**Issue by dogecade** [4Mens4Coding/MathOCRLibrary#4]
Get 3-5 examples ready with pros/cons(2-4 of each) why which is good or bad for image proccessing or ocr.

**Comment by tomas-drasutis** [comment]
GRADIENT DESCENT (NEURAL NETWORK):
https://www.youtube.com/watch?v=IHZwWFHWa-w
https://www.analyticsvidhya.com/blog/2017/03/introduction-to-gradient-descent-algorithm-along-its-variants/
http://wiki.fast.ai/index.php/Gradient_Descent#Algorithm

+Nesudėtinga panaudoti  (bent jau taip daug kur sako)
+Apsimokius tikslumas 96% - 98%
 
-Dėl to kaip veikia yra didelė tikimybė, kad padavus kokią nors nesąmoningą nuotrauką sugražins užtikrintą atsakymą
-Remiasi globalaus minimumo ieškojimu, bet tai gali būti nesėkminga, nes rasim tik lokalų minimumą
-Sunku atitaikyti learning rate, kad būtų optimaliausias ir naudingiausias
